MOBILE_API.xml

```diff
@@ -871,11 +897,11 @@
         <element name="locationImage" since="4.0">
             <description>The optional image of a destination / location</description>
         </element>
-        
+
         <element name="alertIcon" since="6.0">
             <description>The image field for Alert</description>>
         </element>
-        
+
```

Comment:
Not sure if this delta is needed

Reply:
removed in 086a52f7d5dcd3505f62478018294a3bd5165a4f